modular_skyrat/modules/medical/code/cargo/packs.dm

```diff
@@ -0,0 +1,42 @@
+/datum/supply_pack/science/chilled_hercuri
+	name = "Chilled Hercuri Pack"
+	desc = "Contains 2 pre-chilled bottles of hercuri, 100u each. Useful for dealing with severely burnt synthetics!"
+	cost = CARGO_CRATE_VALUE * 2.5
+	contains = list(/obj/item/reagent_containers/spray/hercuri/chilled = 2)
+
+	crate_type = /obj/structure/closet/crate/medical
+	crate_name = "chilled hercuri crate"
+
+	access_view = FALSE
+	access = FALSE
+	access_any = FALSE
+
+/datum/supply_pack/science/synth_treatment_kit
+	name = "Synthetic Treatment Kit"
+	desc = "Contains a treatment kit for synthetic lifeforms, filled to the brim with everything you need to treat an inorganic wound!"
+	cost = CARGO_CRATE_VALUE * 5 // you pay a premium for getting everything all at once
```

Comment:
Usually it's the other way around, you pay less for buying in bulk

Reply:
true ig